XMonad/Hooks/DynamicStatusBarConfigs.hs

```diff
@@ -0,0 +1,125 @@
+{-# LANGUAGE TupleSections #-}
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  XMonad.Hooks.DynamicStatusBarConfigs
```

Comment:
This looks like a nice and very useful module. It deserves a better name than this. :-)
(I'll keep it in mind and if I can think of something, I'll let you know.)

Reply:
I'm... bad at naming things